junit5-extension/src/main/java/io/kroxylicious/testing/kafka/junit5ext/Topic.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.testing.kafka.junit5ext;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Signals the wish for a topic to be created and injected into a test as either test parameter
+ * or field.
+ * {@see KafkaClusterExtension}
+ */
+public interface Topic {
+    @NonNull
+    String name();
```

Comment:
I did think about exposing the topicUuid here.  I decided, YAGNI.

Reply:
Yeah, I think YAGNI is the right call.

It will be very cheap to add if we come up with a motivation for exposing both so lets leave it out for now.